fix.py

```diff
@@ -42,42 +42,45 @@ def get_game_id() -> str:
 def get_game_name() -> str:
     """Trys to return the game name from environment variables"""
     pfx = os.environ.get('WINEPREFIX') or protonmain.g_session.env.get('WINEPREFIX')
-    script_dir = os.path.dirname(os.path.abspath(__file__))
+    script_dir = os.path.dirname(os.path.abspath(__file__))  # noqa: PTH100, PTH120
 
     if os.environ.get('UMU_ID'):
-
-        if os.path.isfile(f'{pfx}/game_title'):
-            with open(f'{pfx}/game_title', encoding='utf-8') as file:
+        if os.path.isfile(f'{pfx}/game_title'):  # noqa: PTH113
+            with open(f'{pfx}/game_title', encoding='utf-8') as file:  # noqa: PTH123
                 return file.readline()
 
         umu_id = os.environ['UMU_ID']
         store = os.getenv('STORE', 'none')
-        csv_file_path = os.path.join(script_dir, 'umu-database.csv')
+        csv_file_path = os.path.join(script_dir, 'umu-database.csv')  # noqa: PTH118
 
         try:
-            with open(csv_file_path, newline='', encoding='utf-8') as csvfile:
+            with open(csv_file_path, newline='', encoding='utf-8') as csvfile:  # noqa: PTH123
                 csvreader = csv.reader(csvfile)
```

Comment:
The CSV is a database for us so we handle the CSV by assuming it's very large.

TODO: Refactor to prefer memory mapping the CSV.